artemis-server/src/main/java/org/apache/activemq/artemis/core/server/impl/QueueImpl.java

```diff
@@ -2712,6 +2714,11 @@ private Message makeCopy(final MessageReference ref, final boolean expiry) throw
    private Message makeCopy(final MessageReference ref,
                             final boolean expiry,
                             final boolean copyOriginalHeaders) throws Exception {
+      if (ref == null) {
+         ActiveMQServerLogger.LOGGER.nullRefMessage();
+         NULL_REF.createException("Reference to message is null");
```

Comment:
didn't you miss throw NULL_REF here?

Reply:
@stanlyDoge what was your intention here? this last expression here is pretty much dead code.. you are creating an exception and doing nothing with it... the previous logging would work.. but the exception here is dead code.

Reply:
Yeah, i forgot to throw that. I am throwing ActiveMQNullRefException, which under the hood does super(ActiveMQExceptionType.NULL_REF). Is that ok?